sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilters.scala

```diff
@@ -270,6 +272,29 @@ private[parquet] class ParquetFilters(pushDownDate: Boolean) {
       case sources.Not(pred) =>
         createFilter(schema, pred).map(FilterApi.not)
 
+      case sources.StringStartsWith(name, prefix) if canMakeFilterOn(name) =>
```

Comment:
What do you think about adding a configuration to control this and set it true by default? It's basically dependent on an user defined predicate we manually wrote here.